tools/pubsub2inbox/output/mail.py

```diff
@@ -0,0 +1,380 @@
+from .base import Output
```

Comment:
This file is huge. It is very hard to follow. Some of the contents of the `if` clausules could be moved to their own functions, and the code would be more legible and easier to follow.

Reply:
It's a bit large, but I wouldn't go as far as say it's huge. I'm planning to refactor this as part of next updates that will also include some new features.